notebooks/github_pr_creator_agent.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>Let's add a description before this, explaning what we're doing. Somethinglike:</p><p> </p><h2>GitHub Issue Resolver </h2><p>First, we'll create a GitHub issue resolver agent, following the steps in this recipe: Build a GitHub Issue Resolver Agent(<a href="https://haystack.deepset.ai/cookbook/github_issue_resolver_agent" target="_blank">https://haystack.deepset.ai/cookbook/github_issue_resolver_agent</a>) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/183/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
added